src/swapd/temporal_safety.rs

```diff
@@ -49,11 +49,17 @@ impl TemporalSafety {
         self.final_tx(lock_confirmations, Blockchain::Bitcoin)
             && lock_confirmations > (self.cancel_timelock - self.race_thr + 1)
     }
+    pub fn blocks_until_stop_funding(&self, lock_confirmations: u32) -> i64 {
```

Comment:
Maybe add a comment so we have a quick understanding without doing the math :p

EDIT: I guess the name of the function is self-explanatory but other fn have comments so... ;)

Reply:
1208d67dd0b8c52ebbffc2ff8d9a0b434b2714bf